### PR TITLE
sci-electronics/fritzing: Version bump to 0.9.10

### PR DESCRIPTION
**Pull request by flint2**

### CODE REVIEW

File: sci-electronics/fritzing/Manifest [hunk 1]
```diff
@@ -1,2 +1,4 @@
...
 DIST fritzing-0.9.4.tar.gz 11430102 BLAKE2B 20232b3c33308edcab188c61597f87d8a21d9b786eeab4e398c1add6e263562d1b0df95634850d5271decfa380d7b2b0c18c7f19f8b8544f330687e4cffa3700 SHA512 b148d8b93d2e452c975ca3acfd2ad5de4259f99da69c5890ebeea30730674526d480ee3830f61f6057e83b7720f07fce80ef3a74d4b718be691a92d656b755e0
+DIST fritzing-parts-0.9.10.tar.gz 17662003 BLAKE2B a67f4f1297793131d25e6b9c853e987fc0c5ecebd5f57b3675b7f9c4539f867ad18fc6e854c7a2bef1669e05dbc13dcc93df9b9befb8991ca64b2bc3e2a71b84 SHA512 f6f6c396eaf0b61dd56ece22f6bb8fce4369503729fd8f5550643e74a60aaad75abce9e391d63d52f12e4f3420296791fdf9832f7c8507dad2cf623e1bd8340f
 DIST fritzing-parts-0.9.4.tar.gz 17662003 BLAKE2B a67f4f1297793131d25e6b9c853e987fc0c5ecebd5f57b3675b7f9c4539f867ad18fc6e854c7a2bef1669e05dbc13dcc93df9b9befb8991ca64b2bc3e2a71b84 SHA512 f6f6c396eaf0b61dd56ece22f6bb8fce4369503729fd8f5550643e74a60aaad75abce9e391d63d52f12e4f3420296791fdf9832f7c8507dad2cf623e1bd8340f
```
Comment (juippis) [inline sci-electronics/fritzing/Manifest:4]: How come these are identical?

File: sci-electronics/fritzing/fritzing-0.9.10.ebuild [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline sci-electronics/fritzing/fritzing-0.9.10.ebuild:4]: Always aim to use the latest available EAPI, 8 should work here.

File: sci-electronics/fritzing/metadata.xml [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
 <pkgmetadata>
-	<!-- maintainer-needed -->
+    <maintainer type="person" proxied="yes">
+        <email>gjoandet@gmail.com</email>
+        <name>Guillermo Joandet</name>
+    </maintainer>
+    <maintainer type="project" proxied="proxy">
+        <email>proxy-maint@gentoo.org</email>
+        <name>Proxy Maintainers</name>
+    </maintainer>
```
Comment (juippis) [inline sci-electronics/fritzing/metadata.xml:11]: Please fix the CI issue this spawned.

File: sci-electronics/fritzing/fritzing-0.9.10.ebuild [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+		sed -e "/PKGCONFIG/s/quazip1-qt5/quazip/" -i phoenix.pro || die
+	fi
+
+	# Get a rid of the bundled libs
+	# Bug 412555 and
+	# https://code.google.com/p/fritzing/issues/detail?id=1898
+	rm -r src/lib/quazip/ pri/quazip.pri || die
+
+	# Fritzing doesn't need zlib
+	sed -i -e 's:LIBS += -lz::' -e 's:-lminizip::' phoenix.pro || die
```
Comment (juippis) [inline sci-electronics/fritzing/fritzing-0.9.10.ebuild:63]: This doesn't seem to do anything anymore.
```
 * SED: some expressions in the following did not cause any changes
 *     sed -i -e 's:LIBS += -lz::' -e 's:-lminizip::' phoenix.pro || die;
 * no-op: -e s:-lminizip::
```

**Comment by flint2** [comment]
0.9.10 never released source code, bumped by mistake

**Pull request closed without merging**